buildbot_nix/__init__.py

```diff
@@ -62,7 +62,9 @@
 from .oauth2_proxy_auth import OAuth2ProxyAuth
 from .projects import GitBackend, GitProject
 
-SKIPPED_BUILDER_NAMES = [f"skipped-builds-{n}" for n in range(cpu_count())]
+SKIPPED_BUILDER_NAMES = [
+    f"skipped-builds-{n:03}" for n in range(int(cpu_count() * 0.25))
```

Comment:
Since it's partially io-bound, we can probably have a few more workers than cores.
Also we shouldn't go down to 0 if there someone has 1-3 cores.

```suggestion
    f"skipped-builds-{n:03}" for n in range(int(max(4, int(cpu_count() * 0.25))))
```